README.md

```diff
@@ -1,32 +1,60 @@
 # Superformula Mobile Developer Coding Test
 
-Make sure you read **all** of this document carefully, and follow the guidelines in it.
+## Dependencies
 
-## Requirements
+- bloc
+- dio
+- flutter_bloc
+- freezed
+- get_it
+- intl
+- json_serializable
+- qr_code_scanner
+- qr_flutter
+- bloc_test
+_ build_runner
+_ mocktail
 
-There is only one test here currently, please review and get back to us.
 
-## What We Care About
+## Run the project
 
-Use any libraries that you would normally use if this were a real production App. Please note: we're interested in your code & the way you solve the problem, not how well you can use a particular library or feature.
+This project contains 3 flavors:
 
-_We're interested in your method and how you approach the problem just as much as we're interested in the end result._
+- development
+- staging
+- production
```

Comment:
question: I know these flavors come out of the box with the `very good cli` but are we using them for anything here?

Reply:
Based on the feedback given , in [this commit](https://github.com/Superformula/mobile-test/pull/55/commits/5e97f7aa879950416017ad86b32ddf955772435d) it's implemented the feature of change color for every flavor.